lib/models/App.js

```diff
@@ -227,10 +155,13 @@ class App {
    */
 
   loadModels() {
```

Comment:
see if we can extract these similar load models, services call into a loop and open up an interface for user to override